spec/controllers/api/v1/reviews_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ReviewsController, type: :controller do
+  let!(:u1) { User.create(email: 'joe@joe.com', password: 'phillyphilly', display_name: 'joe') }
+  let!(:first_trip) { Trip.create(name: 'Liberty Bell', description: 'Cool trip.') }
+  let!(:second_trip) { Trip.create(name: 'Jersey Shore', description: 'Bad trip.') }
+  let!(:r1) { Review.create(user_id: u1.id, trip_id: first_trip.id, rating: 1, body: 'Great') }
+  # let!(:r2) { Review.create(user_id: user.id, trip_id: first_trip.id, rating: 4) }
+  # let!(:r3) { Review.create(user_id: user.id, trip_id: second_trip.id, rating: 3) }
+
+  describe 'POST#create' do
+    it 'creates a new review' do
+      post_json = {
+        review: { rating: 4, body: 'Great' },
+        user_id: u1.id, trip_id: first_trip.id
+      }
+
+      prev_count = Review.count
+      post(:create, params: post_json)
+      expect(Review.count).to eq(prev_count + 1)
+    end
+
+    it 'returns the json of the newly posted trip' do
+      post_json = {
+        review: { rating: 4, body: 'Great' },
+        user_id: u1.id, trip_id: first_trip.id
+      }
+
+      post(:create, params: post_json)
+      returned_json = JSON.parse(response.body)
+      expect(response.status).to eq 200
+      expect(response.content_type).to eq 'application/json'
+
+      expect(returned_json).to be_kind_of(Hash)
+      expect(returned_json).to_not be_kind_of(Array)
```

Comment:
You don't really need this check, there's no risk of this coming back as an array.